recipes/yt/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: windows builds are disabled for now
+:: "%PYTHON%" setup.py install --single-version-externally-managed --record=record.txt
+:: if errorlevel 1 exit 1
```

Comment:
@pelson said he could keep it. So, was just making sure it was commented at least. Otherwise I would have said the same.